From d0a6f3e851042d3e78cec6470083bdcf8f5bb8a4 Mon Sep 17 00:00:00 2001
From: Huaming <jianghuaming.jhm@linux.alibaba.com>
Date: Fri, 21 Jun 2024 14:31:14 +0800
Subject: [PATCH 188/400] defconfig:th1520: enable cma config

1.enable CMA config to support cma pool
2.enable CMA DMA config for "share dma pool"

Signed-off-by: Huaming <jianghuaming.jhm@linux.alibaba.com>
Change-Id: I29db09c7b8ff8ddfaed8dff89ed0e4c6dfd56812
---
 arch/riscv/configs/th1520_defconfig | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

diff --git a/arch/riscv/configs/th1520_defconfig b/arch/riscv/configs/th1520_defconfig
index 4ff862db8bc3..40b65c9ab7ad 100644
--- a/arch/riscv/configs/th1520_defconfig
+++ b/arch/riscv/configs/th1520_defconfig
@@ -295,3 +295,21 @@ CONFIG_GPIO_SYSFS=y
 CONFIG_NEW_LEDS=y
 CONFIG_LEDS_CLASS=y
 CONFIG_LEDS_GPIO=y
+# CONFIG_LOCALVERSION_AUTO is not set
+CONFIG_MEMORY_ISOLATION=y
+CONFIG_CONTIG_ALLOC=y
+CONFIG_CMA=y
+# CONFIG_CMA_DEBUG is not set
+CONFIG_CMA_DEBUGFS=y
+# CONFIG_CMA_SYSFS is not set
+CONFIG_CMA_AREAS=7
+CONFIG_DMA_CMA=y
+#
+# Default contiguous memory area size:
+#
+CONFIG_CMA_SIZE_MBYTES=32
+CONFIG_CMA_SIZE_SEL_MBYTES=y
+# CONFIG_CMA_SIZE_SEL_PERCENTAGE is not set
+# CONFIG_CMA_SIZE_SEL_MIN is not set
+# CONFIG_CMA_SIZE_SEL_MAX is not set
+CONFIG_CMA_ALIGNMENT=8
-- 
2.43.0

